mood_for_food/main.js

```diff
@@ -0,0 +1,96 @@
+// Get elements
+
+const BtnDish = document.getElementById("btn-dish");
+const BtnReset = document.getElementById("btn-reset");
+const MyHeader = document.getElementById("my-header");
+const defaultDiv = document.getElementById("default-div");
```

Comment:
Const variable declarations should be all caps (as in line 9). Source: [Google JavaScript Style guide](https://google.github.io/styleguide/javascriptguide.xml)

Reply:
- Removed MyHeader 
- Replaced "BtnDish" and "BtnReset" to "btnDish" and "btnReset" (camel case as they are ids)